app/src/services/exercise.services.ts

```diff
@@ -2,31 +2,38 @@ import axios from "axios";
 
 // Interfaces
 import { Exercise } from "../interfaces/Exercise";
+const BACKEND_URL = import.meta.env.VITE_BACKEND_URL;
 
+/* Bearer is not implemnted in backend, due to content creators not existing yet */
 
-// Send the info to exercise service
-const addExercise = async (props: Exercise, token: string, sid: string | null | undefined) => {
-  const response = await axios.post(
-    `http://127.0.0.1:8888/api/sections/${sid}/exercises`,
+// Send the info to exercise service 
+const addExercise = async (props: any, token: string, sid: any) => {
+  return await axios.post(
+    `${BACKEND_URL}/api/exercises/create/${sid}`,
     props,
     { headers: { Authorization: `Bearer ${token}` } }
-  )
-
-  return response.data
-};
+  );
+}
 
 // Send the info to exercise service
-const saveExercise = async (props: any, token: string) => {
-  const response = await axios.put(
-    `http://127.0.0.1:8888/api/exercises/${props.id}`,
+const updateExercise = async (props: any, token: string, eid: any) => {
+  const response = await axios.post(
+    `${BACKEND_URL}/api/exercises/update/${eid}`,
```

Comment:
Do not have actions in the endpoint

should be api/exercises/:id

Reply:
Altso, should it be PATCH instead of POST?

Reply:
Thanks for the info, we have fixed it❤